widgets/utils.py

```diff
@@ -33,6 +33,12 @@ def __init__(self, min_val: int, max_val: int, step: int, value: int):
         self.setRange(min_val, max_val)
         self.setSingleStep(step)
         self.setValue(value)
+    
+    def currentText(self) -> str:
+        return str(self.value())
+        
+    def setCurrentText(self, value: str):
+        self.setValue(int(value))
```

Comment:
These methods are not used. If they're needed for anything else, I would include them in a different PR.